compiler/ast/ast.go

```diff
@@ -16,53 +16,79 @@ import (
 // license that can be found in the LICENSE file.
 
 // Op is the interface implemented by all AST operator nodes.
+
+type Node interface {
+	Pos() int // Position of first character belonging to the node.
+	End() int // Position of first character immediately after the node.
+}
+
 type Op interface {
```

Comment:
Move the Op doc comment so it remains adjacent to the Op declaration.

Reply:
But also, why not use compiler/ast/zed.Node instead of declaring this?

Reply:
It just seems weird that Node is part of astzed package, where as ast is the main package.

Reply:
Like at some point we'll have a node Walk function if we want to do things like formatting and it seems weird to me that this would use `astzed.Node`.